mne_sandbox/preprocessing/_sns.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we really need the author names? They are very rarely updated. I think we should get rid of them personally.

The doctsring seems overkill since there is only one class in here, but I'll put it in for consistency.